ops/scripts/batch-submitter.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+set -e
+
+RETRIES=${RETRIES:-40}
+
+if [[ ! -z "$URL" ]]; then
+    # get the addrs from the URL provided
+    ADDRESSES=$(curl --fail --show-error --silent --retry-connrefused --retry $RETRIES --retry-delay 5 $URL)
```

Comment:
Something that we could do that would be a nice to have to eliminate one extra config option would be to only pass in the address manager address and have the batch submitter resolve both the ctc and scc. It could also resolve `OVM_Sequencer` and `OVM_Proposer` to make sure that its own keys match these values, proposer is the entity that has the scc key and sequencer is the entity with the ctc key

Reply:
previously we discussed removing the address resolution and passing in the addresses manually. i'm open do doing this if it's what we want, we should get some input @karlfloersch